scripts/lowStorageAlert/impl-lowStorageAlert.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT license. See LICENSE file on the project webpage for details.
+
+set -x
+
```

Comment:
nit: you may want to put in some comments explaining purpose of this specific script and usage